aws-cpp-sdk-transfer/source/transfer/S3FileRequest.cpp

```diff
@@ -75,6 +75,11 @@ bool S3FileRequest::IsDone() const
 {
     return m_isDone.load();
 }
+    
+bool S3FileRequest::IsCancelled() const
```

Comment:
I couldn't find code that was handling the cancel: will it clean up and delete the file *or* merely stop the upload?